<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KarsilastirmaWidget</class>
 <widget class="QWidget" name="KarsilastirmaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="karsilastirma_ana_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>680</width>
     <height>580</height>
    </rect>
   </property>
  <property name="styleSheet">
   <string notr="true">#karsilastirma_ana_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
}
#sistem_karsilastirma_baslik {
    font-size: 22px;
    font-weight: bold;
    color: #ffffff;
    padding-bottom: 15px;
    border-bottom: 1px solid #3c404d;
    margin-bottom: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <widget class="QLabel" name="sistem_karsilastirma_baslik">
     <property name="text">
      <string>Sistem Karşılaştırması</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="arama_kutusu">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #363a45;
    border: 1px solid #3c404d;
    border-radius: 5px;
    padding: 5px;
    color: white;
}
QLineEdit:focus {
    border: 1px solid #4facfe;
}</string>
       </property>
       <property name="placeholderText">
        <string>Kullanıcı Ara...</string>
       </property>
       <property name="icon" stdset="0">
        <iconset>
         <normaloff>icons/search-xxl.png</normaloff>icons/search-xxl.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QFrame" name="sizin_sisteminiz_frame">
       <property name="styleSheet">
        <string notr="true">background-color: transparent; border-radius: 8px; padding: 10px; border: 1px solid #3c404d;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="sizin_sisteminiz_baslik">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Sizin Sisteminiz&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QLabel" name="sizin_sisteminiz_detay">
           <property name="text">
            <string>Sistem Taranmadı</string>
           </property>
          </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="rakip_sistem_frame">
       <property name="styleSheet">
        <string notr="true">background-color: transparent; border-radius: 8px; padding: 10px; border: 1px solid #3c404d;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="rakip_sistem_baslik">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Rakip Sistem&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="sizin_sisteminiz_label_alt">
     <property name="styleSheet">
      <string notr="true">font-size: 14px; font-weight: bold; color: #4facfe; margin-top: 10px; margin-bottom: 5px;</string>
     </property>
     <property name="text">
      <string>Rakip Sistemler</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="sistem_listesi">
     <property name="styleSheet">
      <string notr="true">QListWidget {
    background-color: #2b2e38;
    border: none;
    color: #e0e0e0;
    font-family: 'Segoe UI', sans-serif;
    font-size: 13px;
    outline: none;
}
QListWidget::item {
    padding: 12px;
    border-bottom: 1px solid #3c404d;
}
QListWidget::item:hover {
    background-color: #363a45;
    color: #ffffff;
}
QListWidget::item:selected {
    background-color: #3c404d;
    color: #4facfe;
}
QScrollBar:vertical {
    border: none;
    background: #2b2e38;
    width: 10px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: #3c404d;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #4facfe;
}
QScrollBar:add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar:add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="karsilastir_buton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Karşılaştır</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <widget class="QFrame" name="detayli_sonuclar_frame">
  <property name="geometry">
   <rect>
    <x>700</x>
    <y>10</y>
    <width>310</width>
    <height>580</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#detayli_sonuclar_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
}
#detayli_sonuclar_baslik {
    font-size: 20px;
    font-weight: bold;
    color: #ffffff;
    padding-bottom: 15px;
    border-bottom: 1px solid #3c404d;
    margin-bottom: 15px;
}</string>
  </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QLabel" name="detayli_sonuclar_baslik">
      <property name="text">
       <string>Detaylı Sonuçlar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="detayli_grafik_tutucu" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="placeholder_detayli_grafik">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 10pt;</string>
       </property>
       <property name="text">
        <string>Grafik Alanı (Sütun Grafik)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QWidget" name="sizin_puan_kutusu" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #007bff; border-radius: 8px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QLabel" name="sizin_toplam_puan_baslik">
           <property name="text">
            <string>Sizin Toplam Puanınız</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizin_puan_deger">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>42500 pts</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rakip_puan_kutusu" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #495057; border-radius: 8px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="rakip_toplam_puan_baslik">
           <property name="text">
            <string>Rakip Toplam Puanı</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rakip_puan_deger">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>45000 pts</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
